kustomization.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+images:
+# Controller
+- name: objectstorage-controller
+  newName: quay.io/containerobjectstorage/objectstorage-controller
+  newTag: latest
+# Sidecar
+- name: sample-driver
+  newName: quay.io/containerobjectstorage/sample-driver
+  newTag: latest
+- name: object-storage-sidecar
+  newName: quay.io/containerobjectstorage/object-storage-sidecar
+  newTag: latest
+
+resources:
+# CRDs
+- https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-api/master/crds/objectstorage.k8s.io_bucketaccessclasses.yaml
+- https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-api/master/crds/objectstorage.k8s.io_bucketaccesses.yaml
+- https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-api/master/crds/objectstorage.k8s.io_bucketaccessrequests.yaml
+- https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-api/master/crds/objectstorage.k8s.io_bucketclasses.yaml
+- https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-api/master/crds/objectstorage.k8s.io_bucketrequests.yaml
+- https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-api/master/crds/objectstorage.k8s.io_buckets.yaml
+# Controller
+- manifests/ns.yaml
+- manifests/sa.yaml
+- manifests/rbac.yaml
+- manifests/deployment.yaml
+# Sidecar
+- https://raw.githubusercontent.com/container-object-storage-interface/cosi-provisioner-sidecar/master/examples/object-storage-sidecar.yaml
```

Comment:
I dont believe we need the raw url. I think you should be able to simply use github.com/kubernetes-sigs/container-object-storage-interface-api/master/crds/objectstorage*

Reply:
I agree that we do not need to version but we should not always pull from master.

Reply:
@wlan0 @brahmaroutu 

It seems like I will have to keep the raw URL here. I am getting following error when I use the direct URL for the file in repo

```
Error: accumulating resources: 2 errors occurred:
	* accumulateFile error: "accumulating resources from 'https://github.com/kubernetes-sigs/container-object-storage-interface-api/blob/master/crds/objectstorage.k8s.io_bucketaccessclasses.yaml': YAML file [https://github.com/kubernetes-sigs/container-object-storage-interface-api/blob/master/crds/objectstorage.k8s.io_bucketaccessclasses.yaml] encounters a format error.\nerror converting YAML to JSON: yaml: line 91: mapping values are not allowed in this context\n"
	* accumulateDirector error: "couldn't make target for path '/private/var/folders/75/l_gy1x_x6yx8vf6dfvg6zx8w0000gn/T/kustomize-974451088/repo': unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory '/private/var/folders/75/l_gy1x_x6yx8vf6dfvg6zx8w0000gn/T/kustomize-974451088/repo'"
```

Reply:
I see, look like we need to only fetch file content.